kernel/src/Boot/Patches/Patches672.cpp

```diff
@@ -13,48 +17,49 @@ void Mira::Boot::Patches::install_prerunPatches_672()
 	// Use "kmem" for all patches
 	uint8_t *kmem;
 
-	// Patch dynlib_dlsym
-	kmem = (uint8_t*)&gKernelBase[0x1D895A];
-	kmem[0] = 0xE9;
-	kmem[1] = 0xC7;
-	kmem[2] = 0x01;
-	kmem[3] = 0x00;
-	kmem[4] = 0x00;
+	// Enable UART
+	kmem = (uint8_t *)&gKernelBase[0x01A6EB18];
+	kmem[0] = 0x00;
 
-	// Patch a function called by dynlib_dlsym
-	kmem = (uint8_t*)&gKernelBase[0x0041A2D0];
-	kmem[0] = 0x31; // xor eax, eax
-	kmem[1] = 0xC0;
-	kmem[2] = 0xC3;	// ret
+	// Verbose Panics
+	kmem = (uint8_t *)&gKernelBase[0x002ED33A];
+	kmem[0] = 0x90;
+	kmem[1] = 0x90;
+	kmem[2] = 0x90;
+	kmem[3] = 0x90;
+	kmem[4] = 0x90;
 
-	// Patch sys_mmap
-	kmem = (uint8_t*)&gKernelBase[0x000AB57A];
-	kmem[0] = 0x37; // mov     [rbp+var_61], 33h ; '3'
-	kmem[3] = 0x37; // mov     sil, 33h ; '3'
+	// sceSblACMgrIsAllowedSystemLevelDebugging
+	kmem = (uint8_t *)&gKernelBase[0x00233430];
+	kmem[0] = 0xB8;
+	kmem[1] = 0x01;
+	kmem[2] = 0x00;
+	kmem[3] = 0x00;
+	kmem[4] = 0x00;
+	kmem[5] = 0xC3;
 
-	// patch sys_setuid
-	kmem = (uint8_t*)&gKernelBase[0x0010BED0]; // call    priv_check_cred; overwrite with mov eax, 0
-	kmem[0] = 0xB8; // mov eax, 0
-	kmem[1] = 0x00;
+	kmem = (uint8_t *)&gKernelBase[0x00233BD0];
+	kmem[0] = 0xB8;
+	kmem[1] = 0x01;
 	kmem[2] = 0x00;
 	kmem[3] = 0x00;
 	kmem[4] = 0x00;
+	kmem[5] = 0xC3;
 
-	// patch sys_mprotect
-	kmem = (uint8_t*)&gKernelBase[0x00451DB8]; // jnz     loc_FFFFFFFF82652426; nop it out
```

Comment:
Why was this removed/changed?

Reply:
This is still here it's just line 153 now